foam_controlwrapper/examples/foam_mesh_to_h5cuds.py

```diff
@@ -0,0 +1,14 @@
+"""Example to convert mesh from OpenFoam's format to H5CUDS
+
+"""
+
+from foam_controlwrapper.foam_controlwrapper import FoamControlWrapper
+from simphony.io.h5_cuds import H5CUDS
+
+foam_controlwrapper = FoamControlWrapper()
+name = 'poiseuille'
```

Comment:
`foam`mesh_to_h5cuds.py` is used to generate the H5CUDS-file used in the two examples, right? 

I am trying to run the examples.  Where can i find `poiseuille`?  What are the steps to create `poiseuille.cuds`